specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/disk.json

```diff
@@ -1154,7 +1154,44 @@
           ],
           "x-ms-enum": {
             "name": "DiskCreateOption",
-            "modelAsString": true
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Empty",
+                "name": "Empty",
+                "Description": "Create a new disk with a given size."
+              },
+              {
+                "value": "Attach",
+                "name": "Attach",
+                "Description": "Disk will be attached to a VM."
+              },
+              {
+                "value": "FromImage",
+                "name": "From Image",
+                "Description": "Create a new disk by importing from a PIR image or a customer's user image."
+              },
+              {
+                "value": "Import",
+                "name": "Import",
+                "Description": "Create a new disk by importing from a customer's blob. The blob must be formatted as a VHD."
```

Comment:
Create a disk by importing from a blob specified by a sourceUri in a storage account specified by storageAccountId